### PR TITLE
Add minimum intensity projection (minip) shading to `VolumeVisual`

### PR DESCRIPTION
**Pull request by alisterburt**
Starting gently after our discussion last week, here's a small PR introducing minimum intensity projection shading as a rendering mode for the `VolumeVisual`. This was introduced into napari in napari/napari#1861 and can be seen there, sorry for the wait in pushing upstream!

### CODE REVIEW

File: vispy/visuals/volume.py [hunk 1]
```diff
@@ -378,6 +402,7 @@
 
 frag_dict = {
     'mip': MIP_FRAG_SHADER,
+    'minip' : MINIP_FRAG_SHADER,
```
Comment (djhoese) [inline vispy/visuals/volume.py:405]:
```suggestion
    'minip': MINIP_FRAG_SHADER,
```

**Review by almarklein — COMMENTED** [review]
LGTM!

**Review by djhoese — APPROVED** [review]

**Comment by djhoese** [comment]
@rougier If you have any comments let us know, otherwise I'll merge this tomorrow morning (my morning).

**Comment by djhoese** [comment]
I added a test that tests all available render methods in my #1912 PR. Because of that I won't ask you to add a test for this PR. Since that PR will take a bit to review I'm OK merging this PR and resolving any issues in the future. This PR isn't too complicated that I couldn't fix things if it did fail...I think :wink: 

Thanks @alisterburt

**Comment by alisterburt** [comment]
thanks for adding the test in #1912 @djhoese - much appreciated!

**Pull request merged**